content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 90
+keywords: [security,certificate]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+By default, the DNS certificates used by the webhooks of Galley and the sidecar
+injector are provisioned and managed by Citadel, which is a large component
+that maintains its own signing key and also acts as a CA for Istio.
+
+In certain deployments, you may want to use your own certificate authority
+instead of Citadel. In those cases, Citadel ends up being used strictly for
+its DNS certificate provisioning functionality. Rather than having to deploy
+Citadel at all in this case, you can instead leverage Chiron, a lightweight
+component that signs certificates using the Kubernetes CA APIs without maintaining its own private key.
+
+This task shows how to provision and manage DNS certificates for Istio control
+plane components through Chiron. Using this feature has the following advantages:
+
+* More lightweight than Citadel.
+
+* Unlike Citadel, this feature doesn't require maintaining a private signing key, which enhances security.
+
+* Simplified root certificate distribution to TLS clients. Clients no longer need to wait for Citadel to generate and distribute its CA certificate.
+
+## Before you begin
+
+* Install Istio through `istioctl` with DNS certificates configured.
+The configuration is read when Pilot starts.
+
+{{< text bash >}}
+$ cat <<EOF | istioctl manifest apply -f -
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  values:
+    global:
+      certificates:
+        - secretName: dns.istio-galley-service-account
+          dnsNames: [istio-galley.istio-system.svc, istio-galley.istio-system]
+        - secretName: dns.istio-sidecar-injector-service-account
+          dnsNames: [istio-sidecar-injector.istio-system.svc, istio-sidecar-injector.istio-system]
+EOF
+{{< /text >}}
+
+* Install [`jq`](https://stedolan.github.io/jq/) for validating the results from running the task.
+
+## DNS certificate provisioning and management
+
+Istio provisions the DNS names and secret names for the DNS certificates based on configuration you provide.
+The DNS certificates provisioned are signed by the Kubernetes CA and stored in the secrets following your configuration.
+Istio also manages the lifecycle of the DNS certificates, including their rotations and regenerations.
+
+## Configure DNS certificates
+
+The [`values-istio-dns-cert.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml)
```

Comment:
This is strange. The `values-istio-dns-cert.yaml` file doesn't seem to have anything to do with this task. Shouldn't it be referring to the `IstioControlPlane` CR above?
```suggestion
The `IstioControlPlane` custom resource used to configure Istio in the `istioctl manifest apply` command, above,
```

Reply:
Revised accordingly.